### PR TITLE
Enhancement: Enable header_comment fixer

### PR DESCRIPTION
**Pull request by localheinz**
This PR

* [x] enables the `header_comment` fixer
* [x] runs `make cs`

💁‍♂️ For reference, see https://github.com/FriendsOfPHP/PHP-CS-Fixer/tree/v2.8.2#usage:

>**header_comment**
>
>Add, replace or remove header comment.
>
>Configuration options:
>
>* `commentType` (`'comment'`, `'PHPDoc'`): comment syntax type; defaults to `'comment'`
* `header` (`string`): proper header content; required
* `location` (`'after_declare_strict'`, `'after_open'`): the location of the inserted header; defaults to `'after_declare_strict'`
* `separate` (`'both'`, `'bottom'`, `'none'`, `'top'`): whether the header should be separated from the file content with a new line; defaults to `'both'`

@chartjes @mdwheele 

If you want to enable this, this mostly required input from your side what you would like the header comment to look like.

### CODE REVIEW

**Comment by chartjes** [comment]
Would this mean that it would enforce there being a header docblook in every single file?

**Comment by localheinz** [comment]
@chartjes 

> Would this mean that it would enforce there being a header docblook in every single file?

Exactly!

**Comment by chartjes** [comment]
@localheinz I'm not sure I want that. What is the normal justification for this?

**Comment by localheinz** [comment]
@chartjes 

Ha, no problem, it was just a suggestion!I think most of the open source projects do have a file-level header, though, so one can at least see where the thing is coming from.

As an example, I have worked on projects before where dependencies had been checked in into version control, rather than using `composer`. Thanks to a file-level header it was a bit easier to find out which one it is and then require it via `composer`.

File: .php_cs.dist [hunk 1]
```diff
@@ -1,5 +1,14 @@
 <?php
 
+$header = <<<TXT
+Copyright (c) TBD
```
Comment (localheinz) [inline .php_cs.dist]: @chartjes 

We can easily specify the year here using `date('Y')`, if you like!

Comment (localheinz) [reply .php_cs.dist]: Might just be a bit of an issue when people contribute after midnight in Europe, hehe, when you guys are still preparing to celebrate a new year.

🎉

Comment (chartjes) [reply .php_cs.dist]: @localheinz If we add in `date('Y')` then I am okay with this

File: .php_cs.dist [hunk 2]
```diff
@@ -1,5 +1,16 @@
 <?php
 
+$year = date('Y');
+
+$header = <<<TXT
+Copyright (c) 2013-$year OpenCFP
```
Comment (localheinz) [inline .php_cs.dist]: @chartjes 

Would this be accurate?

**Comment by localheinz** [comment]
Thank you, @chartjes!

**Pull request merged**